test/sagemaker_tests/pytorch/training/integration/sagemaker/test_mnist.py

```diff
@@ -45,6 +45,7 @@ def test_mnist_distributed_cpu(
 @pytest.mark.multinode(2)
 @pytest.mark.integration("smexperiments")
 @pytest.mark.skip_cpu
+@pytest.mark.skip_pt20_cuda121_tests
```

Comment:
Why skip this mnist distributed gpu test for PT 2.0 cu121 containers?

Reply:
they are mnist distributed gpu but they use [pytorchddp backend](https://github.com/aws/deep-learning-containers/pull/3303/files#diff-e6d86929c1a414e5ba1a993d86961f800af3b187367e7a0ae2e80e632e8a2526R161) which calls smddp which is not installed in image, so I have added new [test](https://github.com/aws/deep-learning-containers/pull/3303/files#diff-f7e0e816691c9539d9fb060a393d936f30047890ad9c3e79bf8386262a48a2f3R69) which calls [mpi](https://github.com/aws/deep-learning-containers/pull/3303/files#diff-e6d86929c1a414e5ba1a993d86961f800af3b187367e7a0ae2e80e632e8a2526R159) backend which is without smddp command.